test/timeout.bats

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	helptext=$(crictl runp -h)
+	if [[ "$helptext" != *"cancel-timeout"* ]]; then
+		skip "must have a crictl with the -T option to test CRI-O's timeout handling"
+	fi
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+function create_conmon() {
+	timeout=$1
+	cat > "$TESTDIR"/tmp_conmon << EOF
+#!/bin/sh
```

Comment:
since you use `[[` below, which is a bashism, you need to have `/bin/bash` here.